docs/UserGuide.adoc

```diff
@@ -91,6 +91,50 @@ Expected output:
 
 *   `Entry deleted: 1. Laksa ($10.50) [Lunch][Food]`
 
+// tag::update[]
+=== Updating a Transaction’s details: `update`
+
+Format: `update i/INDEX | PREFIX_LETTER/UPDATED_DETAILS`
+
+****
+* Updates the Transaction's details at the specified `INDEX`.
+* `PREFIX_LETTER` specifies type of detail to be updated, e.g. `n/` for name.
+* `UPDATED_DETAILS` are the new details of type corresponding to `PREFIX_LETTER` that the Transaction is to be updated with.
```

Comment:
1. Adds in what the `INDEX` referring to? index number that is currently shown in the UI?
2. States that `INDEX` must be a positive integer.

Reply:
Noted!